go/README.md

```diff
@@ -0,0 +1,18 @@
+# substrait-go
+
+Experimental Go bindings for [substrait](https://substrait.io).
+
+## Generate from proto files
+
+```bash
+go generate ./substrait
```

Comment:
Can you instead use `buf gen` for this?

You'll need to add these lines to `buf.gen.yaml`:

```yaml
  - name: go
    out: gen/proto/go
```

Reply:
for ease of import paths, is it okay that my change to buf.gen.yaml uses `out: go` so that the import path for Go would be: `github.com/substrait-io/substrait/go/substrait` rather than `github.com/substrait-io/substrait/gen/proto/go/substrait` ?